src/renderer/dx/DxRenderer.hpp

```diff
@@ -9,6 +9,7 @@
 
 #include <dxgi.h>
 #include <dxgi1_2.h>

```

Comment:
Is this still needed if you're importing 1.3 instead?

Reply:
Yes. `dxgi1_3.h` doesn't include `dxgi1_2.h`.

Reply:
Yes but shouldn't the header supercede the other?

Reply:
No because stuff like `IDXGIFactory2` from `dxgi1_2.h` is still being used.

Reply:
Ah ok. I took a look at your link for "What's new" and see we're still very behind. Since this change will break Win7 compatibility completely (unless software rendering is enabled instead), we should file an issue to update to 1.6 (and make use of whatever we need).